docs/configuring-playbook-bridge-appservice-discord.md

```diff
@@ -0,0 +1,20 @@
+# Setting up Appservice Discord (optional)
+
+The playbook can install and configure [matrix-appservice-discord](https://github.com/Half-Shot/matrix-appservice-discord) for you.
+
+See the project's [documentation](https://github.com/Half-Shot/matrix-appservice-discord/blob/master/README.md) to learn what it does and why it might be useful to you.
+
+Setup Instructions: 
+
+loosely based on [this](https://github.com/Half-Shot/matrix-appservice-discord#setting-up)
+
+1. Create a Discord Application [here](https://discordapp.com/developers/applications/me/create).
+2. Retrieve Client ID and Bot token from this Application.
+3. Enable the bridge with "matrix_appservice_discord_enabled: true" in your vars.yml and provide id and token.
```

Comment:
Wrapping code like this in code blocks (using backticks, instead of double-quotes) would be better.

Reply:
changed. https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/105/commits/b5ae0254fd4c8b24106ba9cb61a3fc2846923392